src/wwwroot/controllers/PlanCtrl.js

```diff
@@ -81,6 +83,14 @@
           });
     }
 
+    function getDeliveriesConsumption() {
```

Comment:
I think this function it's not needed, the exact same service call is made in getCurrentPlanInfo, maybe rename that function or add a boolean to check if the deliveries are more than X amount (consider that this call it's going to be really slow).

Reply:
Deliveries implementation will be removed until decide the best approach to retrieve data from db. I will send the next commit without this. Thanks @mvillaseco